pegasus/migrations/117_fix_pd_program_registration_form_data_and_remove_dupes.rb

```diff
@@ -1,23 +0,0 @@
-Sequel.migration do
-  up do
-    registrations_by_application_id = {}
-
-    DB[:forms].where(kind: 'PdProgramRegistration').each do |form|
```

Comment:
we no longer store this data in pegasus forms in 2023, so it is safe for this migration not to try to find and update this data.